Web_Development/FrontEnd/HTML_Quotation_Citation Elements/README.md

```diff
@@ -0,0 +1,134 @@
+# HTML Quotation and Citation Elements
+---
+
+In HTML, to represent quotations and to cite texts, we need to make use of some specific tags to help us distinguish those quoted or cited text from the normal texts that are written.  
+
+## Quotation and Citation Elements in HTML :
+1. **HTML \<abbr> tag.**
+
+    a)This tag is used to highlight an abbreviation.<br><br>
+    b)They can also be used to write short form of a word like- UN(United Nations), ISRO(Indian Space Research Organisation), GWOC(GirlScript Winter of Contributing)       etc.<br><br>
+    c)We use the attribute 'title' along with the tag \<abbr>. This attribute holds the actual meaning of the abbreviation.<br><br>
+    d)The \<abbr> tag should end with its respective closing tag\</abbr>. <br><br>
+    d)In the output, if we hover/ or move out cursor on the underlined abbreviation, the actual full-form of the abbreviation is displayed.
+
+**Example :**
+
+``` HTML
+    <p>The <abbr title="United Nations">UN</abbr> aims to maintain International Peace.</p>
+```
+
+   **Output :**
+
+![](images/abbr.png)
+
+2. **HTML \<q> tag.**
+
+    a) This tag is used to insert quotation marks when required while writing some quotes.<br><br>
+    b) The \<q> tag should end with its respective closing tag\</q>. 
+
+**Example :**
+
+``` HTML
+     <p>Charlie Chaplin once said: 
+     <q>A day without laughter is a day wasted.</q>
+     which is a very correct statement!</p>
+```
+   **Output :**
+
+![](images/q.png)
+
+3. **HTML \<cite> tag.**
+
+    a) As the name suggests the \<cite> tag is used to specify a citation.<br><br>
+    b) It provides reference/ title  to some unique work done.<br><br>
+    c) It identifies the source of a particular work done.<br><br>
+    d) The text written inside this tag is displayed in italics during the output.<br><br>
+    e) The \<cite> tag should end with its respective closing tag\</cite>. 
+
+**Example :**
+
+``` HTML
+     <p><cite>The Maze Runner</cite> by James Dashner.</p>
+```
+   **Output :**
+
+![](images/cite.png)
+
+4. **HTML \<blockquote> tag.**
```

Comment:
If you could make these as 
#### HTML \<blockquote> tag

Reply:
@ghulamyazdani done pls check.